smdebug/_version.py

```diff
@@ -1 +1 @@
-__version__ = "1.0.0"
```

Comment:
nit: Why is it not 1.0.1?

Reply:
1.0.1 is released. Next time when when we create a branch for 1.0.2.  It will automatically inherit this _version.py

Reply:
Alright